dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -81,6 +81,48 @@ CWWKD1005.find.rtrn.err.explanation=The return type of the method is not \
 CWWKD1005.find.rtrn.err.useraction=Refer to the Jakarta Data Javadoc \
  information for a list of valid return types for repository find methods.
 
+CWWKD1006.delete.rtrn.err=CWWKD1006E: The {0} return type of the {1} method \
```

Comment:
CWWKD1006.delete.rtrn.err=CWWKD1006E: The {0} return type of the {1} method of the {2} repository interface is not a valid return type for a delete operation. 

CWWKD1006.delete.rtrn.err.explanation=The return type of the method is not one of the supported return types for a delete operation.

CWWKD1006.delete.rtrn.err.useraction=Update the repository method to use one of the supported return types. Use a return type of void to return no value. Use a return type of long or int for a deletion count. Use a return type of boolean for an indicator of whether any entities were deleted. Use a return type of array, List, or Optional, that is parameterized with either an entity class or a unique identifier class to return the deleted entities or entity IDs.

Reply:
For this message and all of the others reviewed, I am willing to duplicate information into the useraction if you wish, but I will not be removing this information that customer needs from the main error message as has been suggested. Doing so would switch us from having excellent usability to poor usability. It is not acceptable to do that.